source/auth/tests/mongodb-oidc.rst

```diff
@@ -5,31 +5,30 @@ MongoDB OIDC
 Drivers MUST test the following scenarios:
 
 - ``Callback-Driven Auth``
-- ``AWS Automatic Auth``
 - ``Callback Validation``
-- ``Cached Credentials``
 - ``Speculative Authentication``
 - ``Reauthentication``
+- ``Separate Connections Avoid Extra Callback Calls``
 
 
 .. sectnum::
 
-Drivers MUST be able to authenticate using either authentication or provider
-name if there are multiple principals configured on the server.  Note that
-``directConnection=true`` and ``readPreference=secondaryPreferred`` are needed because the server is a secondary on a replica set, on port ``27018``.
+Drivers MUST be able to authenticate against a server configured with either one or two configured identity providers.
```

Comment:
Is it enough to test with either one (pick one), or should it succeed with either (both)?